### ISSUE TITLE
struct comments

### ISSUE THREAD

**Issue by podhmo** [podhmo/reflect-shape#16]
for https://github.com/podhmo/reflect-openapi/issues/56

**Comment by podhmo** [comment]
go-outline's approach
https://github.com/ramya-rao-a/go-outline/blob/master/main.go

**Comment by podhmo** [comment]
use https://github.com/podhmo/commentof

**Comment by podhmo** [comment]
- [x] struct support
- [x] function support
- [x] method support
- [x] see prev datap and next datap
- [x] cache
- [x] integration (function)
- [x] integration (struct)
- [x] remove arglist, comment

**Comment by podhmo** [comment]
go/packages.Load() is too slow..

```console
$ go test -v
=== RUN   TestStruct
--- PASS: TestStruct (7.10s)
=== RUN   TestFunc
--- PASS: TestFunc (0.00s)
=== RUN   TestMethod
=== RUN   TestMethod/pointer
=== RUN   TestMethod/value
--- PASS: TestMethod (0.00s)
    --- PASS: TestMethod/pointer (0.00s)
    --- PASS: TestMethod/value (0.00s)
PASS
ok      github.com/podhmo/reflect-shape/metadata        7.107s
```

**Comment by podhmo** [comment]
fixed by #30
